kstatemachine/src/commonMain/kotlin/ru/nsk/kstatemachine/MetaInfo.kt

```diff
@@ -0,0 +1,9 @@
+package ru.nsk.kstatemachine
+
+sealed interface MetaInfo
+interface UmlMetaInfo: MetaInfo {
+    val umlLabel: String
+}
+fun umlLabel(label: String) = object : UmlMetaInfo {
```

Comment:
we can make UmlMetaInfo a `fun interface`, so this function will be not necessary

it will look like:

```
val metaInfo = UmlMetaInfo { "myLabel" }
```

Reply:
the rest lgtm. please, fix this and I will merge the PR

Reply:
or you suppose that there might be many properies for this class?

Reply:
i just needed label for uml. But you decided to make it extendable. So what is supposed here to be?